index.js

```diff
@@ -0,0 +1,203 @@
+document.addEventListener("DOMContentLoaded", () => {
+  recipeObject.forEach((recipe) => {
+    setRecipe(recipe);
+  });
+});
+
+
+
+function setRecipe(recipe) {
+  setRecipeTitle(recipe);
+  setRecipeImage(recipe);
+  setRecipeDescription(recipe);
+  setIngredients(recipe);
+  setPreparationSteps(recipe);
+}
+
+const recipeObject = [
```

Comment:
`recipeObject` is an array of objects as indicated by the brackets `[...]`. A single objects is written using curly brackets `{...}`. In this case, the  variable name `recipeObject` can be misleading. When you have array variables your typically name it with a plural s at the end like `recipes`:

``` js
const users = [
  {
    username: "Alice",
    email: "alice@email.net",
    kids: ["Bob", "Charlie"],
  },
  {
    username: "Bob",
    email: "bob@email.net",
    kids: []
  },
];

const dog = {
  name: "Fluffy",
  age: 5,
  breed: "Poodle",
};
```

or consider removing the square brackets so it is a single object instead of an array